react/features/always-on-top/ToolbarButton.tsx

```diff
@@ -54,7 +53,6 @@ const ToolbarButton = ({
     return (<div
         aria-disabled = { disabled }
         aria-label = { accessibilityLabel }
-        aria-pressed = { toggled }
```

Comment:
Why dropping this attribute?

Reply:
it was because the previous state was always read as only "pressed" or "not pressed" as the button clicks on it. so i removed the aria-pressed from almost everywhere! anyhow i think where needed we can just add more statefull describtion. therefore, reverted!

Reply:
I agree and understand. Not relevant because buttons are pressed to enable or disable something, except when the buttons themselves are disabled or not available. Let's remove this attribute.

Reply:
On second thought, let's not remove it. We have buttons that open panels like Chat or Participants. This is a case which requires knowing if a button was or wasn't pressed.